compiler/rustc_passes/src/check_attr.rs

```diff
@@ -270,6 +270,16 @@ impl CheckAttrVisitor<'tcx> {
                                 )
                                 .emit();
                         }
+                        if attr.style == AttrStyle::Inner && CRATE_HIR_ID == hir_id {
```

Comment:
```suggestion
                        if CRATE_HIR_ID == hir_id {
```

The attribute style check isn't needed.